static/app/components/events/interfaces/spans/spanBar.tsx

```diff
@@ -874,7 +874,7 @@ class SpanBar extends React.Component<SpanBarProps, SpanBarState> {
               spanBarHatch={!!spanBarHatch}
               style={{
                 backgroundColor: spanBarColour,
-                left: `clamp(0%, ${toPercent(bounds.left || 0)}, calc(100% - 1px))`,
+                left: `min(${toPercent(bounds.left || 0)}, calc(100% - 1px))`,
```

Comment:
I think we'll want to make the corresponding changes in trace view https://github.com/getsentry/sentry/blob/dd3f634012ceb2f97923fcd1999b974316adb4e9/static/app/views/performance/traceDetails/transactionBar.tsx#L352 as well.

Reply:
Fixed in https://github.com/getsentry/sentry/pull/26116/commits/b3e78e20e683e8791a8ece011b0646d9f6215b0e